src/services/loneAntAnimalGameService.ts

```diff
@@ -0,0 +1,184 @@
+import { Ant } from '../models/ant';
+import { AnimalGameService } from './interfaces/animalGameService';
+import { AnimalAction } from '../models/animal-action';
+import { Animal } from '../models/animal';
+import { Board } from '../models/board';
+import { Game } from '../models/game';
+import { Food } from '../models/food';
+
+const FOOD_PERCENTAGE = +(process.env.FOOD_PERCENTAGE || 0.05);
+const loneAntsDirectory = './ants';
+const MAX_ANTS = +(process.env.MAX_ANTS || 10);
+const GRID_WIDTH = +(process.env.GRID_WIDTH || 200);
```

Comment:
Should the common config elements like grid width/height, ticks per sec, etc that would be the same be pulled before and passed into the game service start method or w/e?

Reply:
these should be separate and not really shared. These games will diverge wildly in the future.